apps/prairielearn/src/pages/instructorAssessmentManualGrading/assessmentQuestion/assessmentQuestion.ts

```diff
@@ -216,7 +235,8 @@ router.post(
           update_assigned_grader: 'assigned_grader' in action_data,
           assigned_grader: action_data?.assigned_grader,
         });
-        res.send({});
+        res.send({ success: true });
```

Comment:
Why not send a [`204`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Reference/Status/204) in this case? Seems like the perfect use case for that.